### DISCUSSION TITLE
Cannot merge PRs when merge queue enabled

### DISCUSSION THREAD

**Discussion by ethanmills** [community/community#47743] [Mobile]
### Select Topic Area

Bug

### Body

We recently enabled merge queues on a repo, and I am now unable to merge PRs from the android app. The button reads "Squash and merge", and pressing it shows an error. 

> Something went wrong while executing your query. Please include 'BA5E:1FCF:40ADA97:41A7205:63EF3B89' when reporting this issue.

**Comment by alcere** [comment]
Hi @ethanmills We are deeply sorry for the trouble. Could you share with us the platform and app version you are using? Thanks!

**Comment by ethanmills** [comment]
@alcere Github mobile v.1.100.0 (705) on Android. It's a beta feature, so not a massive deal that it doesn't work yet, but I just wanted to check it was on your radar

**Reply by alcere** [reply]
Thanks for the heads up! we really appreciate it! Can you give a try after updating to 1.101 and let us know the result? Thanks!
